recipe/meta.yaml

```diff
@@ -8,24 +8,24 @@ source:
   sha256: eb4b9c2956e38f89d588a0d0af0e27a03f1e1d2be705a828bbd12cfba0b020fa
 
 build:
-  number: 0 
+  number: 1
   script: pip install --no-deps .
 
 requirements:
   build:
     - python
     - pip
     - zlib 1.2*
-    - libtiff 4*
-    - jpeg 8*
+    - libtiff 4.0.6
+    - jpeg 9*
     - freetype 2.5*
     - tk
   run:
     - python
-    - jpeg 8*
+    - jpeg 9*
     - zlib 1.2*
     - freetype 2.5*
-    - libtiff 4*
+    - libtiff 4.0.6
```

Comment:
> or there is a chance of getting the default channel libtiff 4.0.2 that is compiled against jpeg 8d.

But there should be no such chance - the metadata describes the situation correctly, and conda should be able to resolve it. I agree though, it looks like it isn't: https://github.com/conda-forge/pillow-feedstock/pull/3#issuecomment-214479633

I'm trying to get together a test case for it.

My general advice though is to try to avoid pinning all the way down to patch version unless there is a good reason (like the conda solver is broken).